src/Robot.Drivers/Sonar/HcSr04DistanceSensorDriverSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Robot.Drivers.Sonar
+{
+    public class HcSr04DistanceSensorDriverSettings
+    {
+        /// <summary>
+        /// Unique ID to refer this sensor.
+        /// </summary>
+        public int SensorId { get; set; }
+
+        /// <summary>
+        /// Sensor angle in radians.
+        /// </summary>
+        /// <remarks>Zero is a forward direction of a Robot. Pi/2 is a right side, -Pi/2 is a left side</remarks>
+        public double Angle { get; set; }
+
+        /// <summary>
+        /// GPIO Trigger PIN number.
+        /// </summary>
+        public int TriggerPin { get; set; }
+
+        /// <summary>
+        /// GPIO Echo PIN number.
+        /// </summary>
+        public int EchoPin { get; set; }
```

Comment:
Probably we dont need it now, but I know that this sensor can be used sharing the same pin as trigger and echo> I did that with arduino before and works fine (and saves a pin per sensor). Not sure if the implementation here allows us to do that, but at some point later as an improvement we could try that

Reply:
Do we really have to conserve PINs? I thought there's enough of them... Anyway, let's consider it later. If we need, we could even share one echo pin for all the sonars 😊